code/zFulpstationCode/fulp_overwrite_vars.dm

```diff
@@ -446,3 +446,34 @@
 //**** Sleepy Pen for ops ENDS - Glubtok, Jul 2020
 //***********************************************************
 
+
+
+//***********************************************************
+//****UNSTAGRAV HOLODRAGON  PR *****************************
+//***********************************************************
+
+/obj/item/clothing/under/rank/engineering/atmospheric_technician
+	desc = "It's a jumpsuit worn by atmospheric technicians. It is protected from fire, not sure about you tho."
+	name = "atmospheric technician's jumpsuit"
+	icon_state = "atmos"
+	inhand_icon_state = "atmos_suit"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 10, "fire" = 0, "acid" = 20)
```

Comment:
why'd you nerf these armor values down from `armor = list(MELEE = 0, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 0, RAD = 10, FIRE = 60, ACID = 20)` to this?